app/views/sufia/homepage/_home_header.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class="home_call_action col-xs-12 col-sm-4 pull-right">
-  <% if can? :view_share_work, GenericWork %>
+  <% if Sufia.config.always_display_share_button || can_ever_create_works? %>
```

Comment:
Not a fan of this logic; Where is `can_ever_create_works?` defined? It makes sense to me to have a method that reflects this particular bit of logic?